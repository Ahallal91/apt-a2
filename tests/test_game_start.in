BRLURBBBBRYUYBLLBBRYRBLYLUURBUUYBYRRRUBUYUBBBRUUYBRULRYUBURRYUYYRUBLYUUYLLRLYRRLYYYLLYLLLLLBULBURLRY
P1
P2
