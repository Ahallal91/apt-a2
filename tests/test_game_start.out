=== Round 1 ===

Factories:
0: F 
1: B R L U 
2: R B B B 
3: B R Y U 
4: Y B L L 
5: B B R Y 

Score for P1: 0
Mosaic for P1:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
6: broken |

Score for P2: 0
Mosaic for P2:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
6: broken |

